tbx/core/blocks.py

```diff
@@ -48,6 +48,33 @@ def text(self):
             return page.title
 
 
+class StyledPageLinkBlock(blocks.StructBlock):
+    """
+    A styled page link block with optional title and style choices.
+    """
+
+    class StyleChoice(models.TextChoices):
+        CHARITY = "charity", "Charity"
+        PUBLIC = "public", "Public"
+        WAGTAIL = "wagtail", "Wagtail"
```

Comment:
suggestion: use the colour theme name a la https://github.com/torchbox/torchbox.com/blob/480360cc67dfbe2e822900ff7ba4fa5e6baa731a/tbx/core/blocks.py#L582-L586 (best to move `ColourTheme` to top level and re-use)